packages/dashboards/addon/routes/dashboards/dashboard/view.js

```diff
@@ -60,7 +60,7 @@ export default Route.extend({
    */
   async model(_, transition) {
     const dashboard = this.modelFor('dashboards.dashboard');
-    const filterQueryParams = get(transition, 'queryParams.filters');
+    const filterQueryParams = get(transition, 'to.queryParams.filters');
```

Comment:
```js
const filterQueryParams = transition.to.queryParams.filters;
```

Reply:
I'm going to upgrade all the navi packages to 3.13 next. Hopefully the codemods will get rid of the rest of the `get` and `set` statements